test/tests/time_integrators/actually_explicit_euler_verification/ee-2d-linear.i

```diff
@@ -0,0 +1,100 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = -1
+  xmax = 1
+  ymin = -1
+  ymax = 1
+  nx = 10
+  ny = 10
+  elem_type = QUAD4
+[]
+
+[Functions]
+  [./ic]
+    type = ParsedFunction
+    value = 0
+  [../]
+
+  [./forcing_fn]
+    type = ParsedFunction
+    value = (x+y)
+  [../]
+
+  [./exact_fn]
+    type = ParsedFunction
+    value = t*(x+y)
+  [../]
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+
+    [./InitialCondition]
+      type = FunctionIC
+      function = ic
+    [../]
+  [../]
+[]
+
+[Kernels]
+  [./ie]
+    type = TimeDerivative
+    variable = u
+    lumping = true
+    implicit = true
+  [../]
+
+  [./diff]
+    type = Diffusion
+    variable = u
+    implicit = false
+  [../]
+
+  [./ffn]
+    type = BodyForce
+    variable = u
+    function = forcing_fn
+    implicit = false
+  [../]
+[]
+
+[BCs]
+  active = 'all'
+
+  [./all]
+    type = FunctionDirichletBC
+    variable = u
+    boundary = '0 1 2 3'
+    function = exact_fn
+    implicit = true
+  [../]
+[]
+
+[Postprocessors]
+  [./l2_err]
+    type = ElementL2Error
+    variable = u
+    function = exact_fn
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  scheme = 'explicit-euler'
+  solve_type = 'LINEAR'
```

Comment:
This file has `implicit` flag set properly,  but it should not use `scheme` or `solve_type` and should use:
```
type = ActuallyExplicitEuler
```